R/tax_range_strat.R

```diff
@@ -104,14 +107,18 @@
 #' title(xlab = "Taxon", line = 10.5)
 #'
 #' @export
-tax_range_strat <- function(occdf, name = "genus", level = "bed",
+tax_range_strat <- function(occdf, name = "genus", group = NULL, level = "bed",
```

Comment:
We have a lot of code for examples already for this function - is it worth having an example using group?